ltsp/server/ipxe/55-ipxe.sh

```diff
@@ -43,6 +43,11 @@ ipxe_main() {
         items="${items:+"$items\n"}$(printf "item --key %d %-20s %s" "$((key%10))" "$img_name" "$title")"
         gotos=":$img_name\n$gotos"
     done
+    # FIXME: Alkis, it seems you are using the "r_" prefix here to prevent
+    # naming collisions. However, such collisions still would occure in
+    # /srv/tftp/ltsp, as you are not using a "roots" subdirectory to separate
+    # these namespaces. I have implemented the cmdline_boot_method items without
+    # the prefix in ipxe menu for that reason.
```

Comment:
If an image source exists in /srv/ltsp/chrootX and in /srv/ltsp/images/chrootX.img, then they should contain the same kernel/initrd, which is the one in /srv/tftp/ltsp/chrootX. It shouldn't be considered a collision.

Reply:
:roots remains a little bit of a mystery to me for now.

Do I understand that correctly:
If one has a chroot you would create a squashfs image from it and boot from it by default but also make the actual chroot available to boot as NFS root without the squashfs_over_nfs detour? Possibly in rw mode so you can comfortably run maintenance tasks on it? 

Makes sense.

Would make sense for CMDLINE_BOOT_METHOD either!

But this would mean, a single CMDLINE_BOOT_METHOD config key for ltsp.conf is not enough to accommodate the parallel presentation of a "RO" boot method and a "RW" boot method.

On the other hand, if it were possible to define a CMDLINE_BOOT_METHOD_RO and a 
CMDLINE_BOOT_METHOD_RW or more generically CMDLINE_BOOT_METHOD_[0-9]* all current scenarios could be implemented using theses as well as RBD and anything coming in the future.

Reply:
As mentioned two topics above, a second ltsp.conf option would be needed to make this work for all scenarios:

BOOT_CMDLINE_[0-9]*     (renamed from CMDLINE_BOOT_METHOD)
BOOT_DIR_[0-9]*